Error building Jaguar with Java 9/10 and using JDK Toolchain 6/7

The error is:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:jar (default) on project jaguar2-api: MavenReportException: Error while generating Javadoc:
[ERROR] Exit code: 1 - javadoc: error - invalid flag: -html5
[ERROR]
[ERROR] Command line was: /Library/Java/JavaVirtualMachines/1.6.0.jdk/Contents/Home/bin/javadoc -html5 @options @packages
[ERROR]
```

Probably an issue with the profile precedence after PR #79.

I believe that the `additionalJOption` configuration of profile [toolchain-jdk6](https://github.com/saeg/jaguar2/blob/118cf504c26e74cd06e7c83897625db2dca5db6d/jaguar2-build/pom.xml#L163) and [toolchain-jdk7](https://github.com/saeg/jaguar2/blob/118cf504c26e74cd06e7c83897625db2dca5db6d/jaguar2-build/pom.xml#L181) isn't overriding the same configuration on profile [html5-javadoc](https://github.com/saeg/jaguar2/blob/118cf504c26e74cd06e7c83897625db2dca5db6d/pom.xml#L303-L313)

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -42,3 +42,36 @@ jobs:
     - name: Build
       if: ${{ matrix.java < 8 }}
       run: ./mvnw install -Prelease -Djdk=${{ matrix.java }} -Dbytecode=${{ matrix.java }}
+
+  issue-80:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04]
+        java: [9, 10]
+        jdk: [6, 7]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup JDK toolchain
+      uses: actions/setup-java@v3
+      with:
+        distribution: zulu
+        java-version: ${{ matrix.jdk }}
+        java-package: jdk
+        cache: maven
+        mvn-toolchain-id: ${{ matrix.jdk }}
+    - name: Setup Java
+      uses: actions/setup-java@v3
+      with:
+        distribution: zulu
+        java-version: ${{ matrix.java }}
+        java-package: jdk
+    - name: Setup GPG
+      run: gpg --batch --passphrase '' --quick-gen-key "John Doe <johndoe@x.y>" ed25519 sign 1d
+    - name: Available toolchains
+      run: cat $HOME/.m2/toolchains.xml
+    - name: Java version
+      run: java -version && javac -version
+    - name: Build
+      run: ./mvnw install -Prelease -Djdk=${{ matrix.jdk }} -Dbytecode=${{ matrix.jdk }}

--- a/README.md
+++ b/README.md
@@ -24,6 +24,9 @@
 * Disable PMD type resolution when targeting Java 21 bytecode (
   [#94](https://github.com/saeg/jaguar2/pull/94)
 ).
+* Fix invalid `-html5` Javadoc option when building with Java 9 or 10 and using JDK Toolchain 6 or 7 (
+  [#96](https://github.com/saeg/jaguar2/pull/96)
+).
 
 ### [v0.0.2](https://github.com/saeg/jaguar2/releases/tag/v0.0.2)
 

--- a/jaguar2-build/pom.xml
+++ b/jaguar2-build/pom.xml
@@ -226,11 +226,10 @@
 			<properties>
 				<!--
 					overrides properties possibly defined by
-					jdk12, jdk20 or html5-javadoc profiles
+					jdk12 and jdk20 profiles
 				-->
 				<source>1.6</source>
 				<bytecode>1.6</bytecode>
-				<additionalJOption />
 			</properties>
 		</profile>
 		<profile>
@@ -244,11 +243,10 @@
 			<properties>
 				<!--
 					overrides properties possibly defined by
-					jdk12, jdk20 or html5-javadoc profiles
+					jdk12 and jdk20 profiles
 				-->
 				<source>1.6</source>
 				<bytecode>1.6</bytecode>
-				<additionalJOption />
 			</properties>
 		</profile>
 		<profile>

--- a/pom.xml
+++ b/pom.xml
@@ -268,7 +268,7 @@
 			</properties>
 		</profile>
 		<profile>
-			<id>toolchain-jdk6-versions</id>
+			<id>toolchain-jdk6-parent</id>
 			<activation>
 				<property>
 					<name>jdk</name>
@@ -276,11 +276,12 @@
 				</property>
 			</activation>
 			<properties>
+				<additionalJOption />
 				<maven-pmd-plugin.version>3.13.0</maven-pmd-plugin.version>
 			</properties>
 		</profile>
 		<profile>
-			<id>toolchain-jdk7-versions</id>
+			<id>toolchain-jdk7-parent</id>
 			<activation>
 				<property>
 					<name>jdk</name>
@@ -288,6 +289,7 @@
 				</property>
 			</activation>
 			<properties>
+				<additionalJOption />
 				<maven-pmd-plugin.version>3.13.0</maven-pmd-plugin.version>
 			</properties>
 		</profile>

